Sources/MongoSwift/BSON/BSONValue.swift

```diff
@@ -853,17 +622,56 @@ public struct CodeWithScope: BSONValue, Equatable, Codable, Hashable {
             let scopeDoc = Document(stealing: scopeData)
 
             return self.init(code: code, scope: scopeDoc)
+        })
+    }
+}
+
+/// A struct to represent the BSON Code type.
+public struct Code: BSONValue, Equatable, Codable, Hashable {
```

Comment:
Keeping with the 1:1 mapping between `BSONValue` and `BSON`, we now have separate `Code` and `CodeWithScope` structs.